lib/matplotlib/offsetbox.py

```diff
@@ -1205,6 +1225,7 @@ def __init__(self, arr,
                  **kwargs
                  ):
 
+        OffsetBox.__init__(self)
```

Comment:
Indeed, I think that is the case in matplotlib proper. I know that in mplot3d, I had to be a little careful with construction, though. Given that the mplot3d tests did not fail, though, I am going to assume that all is well there (although it would make sense to double-check that interactivity for mplot3d isn't impacted).